var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -24,24 +24,47 @@
 ##############################################################################
 from spack import *
 
-
-class Scr(Package):
+class Scr(CMakePackage):
     """SCR caches checkpoint data in storage on the compute nodes of a
        Linux cluster to provide a fast, scalable checkpoint/restart
        capability for MPI codes"""
 
-    homepage = "https://computation.llnl.gov/project/scr/"
+    homepage = "http://computation.llnl.gov/projects/scalable-checkpoint-restart-for-mpi"
+
+    ## NOTE: scr-v1.1.8 is built with autotools and is not properly build here.
+    ## scr-v1.1.8 will be deprecated with the upcoming release of v1.2.0
+    # url      = "https://github.com/LLNL/scr/releases/download/v1.1.8/scr-1.1.8.tar.gz"
+    # version('1.1.8', '6a0f11ad18e27fcfc00a271ff587b06e')
```

Comment:
@tgamblin How frequent are cases where a package changes build system and we want to support both? I ask because multiple inheritance won't work out of the box, but we can think of adding a build system class that is a "variant" (in the design pattern sense) and can defer to a later stage the instantiation of its actual type.

For example we can have this "shell" build system and instruct it to turn either in `AutotoolsPackage` or `CMakePackage` after the inspection of the version of the concrete spec.